SwiftWisdom/Core/StandardLibrary/String/String+Indexing.swift

```diff
@@ -14,6 +14,24 @@ extension String {
         return String(substringCharacters)
     }
 
+    /// Returns a substring in the given range.
+    /// If the range is beyond the string's length, returns the original string.
```

Comment:
Does it make sense to return the original string if the `Range` extends past the end of the string but the lower bound isn't zero?  For example, shouldn't

    let hello = "Hello"
    hello[ip_safely: 1...5]

return `ello` rather than `Hello`?